src/main/java/zmaster587/advancedRocketry/asm/ClassTransformer.java

```diff
@@ -746,28 +750,17 @@ public ClassTransformer() {
 						if(numALoadsInARow == 0)
 							firstALoadIndex = i;
 						numALoadsInARow++;
-						if(numALoadsInARow == 2) {
-							pos = ain;
+						if(numALoadsInARow == 4) {
```

Comment:
What's the logic behind checking for 4 GETFIELDs in a row here?

Reply:
Finished in discussion on discord & latest commit